crates/opencascade-sys/src/lib.rs

```diff
@@ -21,7 +21,9 @@ pub mod ffi {
 
         // Runtime
         type Message_ProgressRange;
-        pub fn Message_ProgressRange_ctor() -> UniquePtr<Message_ProgressRange>;
+
+        #[rust_name = "Message_ProgressRange_ctor"]
+        pub fn construct_unique() -> UniquePtr<Message_ProgressRange>;
```

Comment:
Why not go the other way and use `cxx_name` attributes. Since you are writing rust its better to keep the rust side 1:1 no?

Reply:
Oh, good call! I'll make this change.